totalspineseg/inference.py

```diff
@@ -698,5 +704,60 @@ def main():
             },
         )
 
+    if not iso:
```

Comment:
We need to deal with the case when only the step1 is run

Reply:
Yes
I think if the folder doesn't exist it just do nothing. But ill test it

Reply:
I'll add an `if` for it as it is not mice to see something like this (: 
0it [00:00, ?it/s]